01_prepare_host.sh

```diff
@@ -38,20 +38,36 @@ if ! which kustomize 2>/dev/null ; then
     sudo mv kustomize /usr/local/bin/.
 fi
 
-# Download Ironic binary and Ironic inspector image
+# Clean-up any old ironic containers
+for name in ironic ironic-inspector dnsmasq httpd mariadb ipa-downloader; do
```

Comment:
wondering if we should just add a variable with all those services in common.sh since we reuse them in host_cleanup.sh

Reply:
Most of the code you're commenting here is existing, I've just moved it.  I think they're good suggestions but I'm not keen to rewrite everything here as part of this change.

Reply:
Actually I'll fix what I can here

Reply:
of course can be done in follow-ups